sdk/storage/azure-storage-blobs/test/ut/block_blob_client_test.hpp

```diff
@@ -3,6 +3,20 @@
 
 #include <azure/storage/blobs.hpp>
 
+#include <azure/core/platform.hpp>
+
+#if defined(AZ_PLATFORM_POSIX)
+#include <unistd.h>
+#elif defined(AZ_PLATFORM_WINDOWS)
+#if !defined(WIN32_LEAN_AND_MEAN)
+#define WIN32_LEAN_AND_MEAN
+#endif
+#if !defined(NOMINMAX)
+#define NOMINMAX
+#endif
+#include <windows.h>
+#endif
+
```

Comment:
why do we need these?

Reply:
I was getting build errors from a few places when I removed this from `bodystream.hpp`

Reply:
Test code uses the sdk in the same way as customers do. Customers are going to face the same errors as you did, and it'd be bad experience if they have to include these in their file.

Reply:
Good point. Let me circle back to this one a bit later and investigate why it was needed in the first place.